compiler/rustc_parse/messages.ftl

```diff
@@ -309,6 +309,13 @@ parse_inclusive_range_no_end = inclusive range with no end
 parse_incorrect_braces_trait_bounds = incorrect braces around trait bounds
     .suggestion = remove the parentheses
 
+parse_incorrect_restriction = incorrect {$noun} restriction
+    .help = some possible {$noun} restrictions are:
+            `{$keyword}(crate)`: {$adjective} only in the current crate
+            `{$keyword}(super)`: {$adjective} only in the current module's parent
```

Comment:
I'd suggest "parent module" instead of "module's parent"

Reply:
This specific wording comes the existing message from visibility: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=eada4e689cde0f50de7266394ed07e1f